### PR TITLE
HTCONDOR-1626-remove-using-namespace-std

### PR DESCRIPTION
**Pull request by GregThain**
<Insert PR description here, and leave checklist below for code review.>

# HTCondor Pull Request Checklist for internal reviewers

- [ ] Verify that (GitHub thinks) the merge is clean. If it isn't, and you're confident you can resolve the conflicts, do so. Otherwise, send it back to the original developer.
- [ ] Verify that the related Jira ticket exists and has a target version number and that it is correct.
- [ ] Verify that the Jira ticket is in review status and is assigned to the reviewer.
- [ ] Verify that the Jira ticket (HTCONDOR-xxx) is mentioned at the beginning of the title. Edit it, if not
- [ ] Verify that the branch destination of the PR matches the target version of the ticket
- [ ] Check for correctness of change
- [ ] Check for regression test(s) of new features and bugfixes (if the feature doesn't require root)
- [ ] Check for documentation, if needed
- [ ] Check for version history, if needed
- [ ] Check BaTLab dashboard for successful build (https://batlab.chtc.wisc.edu/results/workspace.php) and test for either the PR or a workspace build by the developer that has the Jira ticket as a comment.
- [ ] Check that each commit message references the Jira ticket (HTCONDOR-xxx)

## After the above
- Hit the merge button if the pull request is approved and it is not a security patch (security changes require 2 additional reviews)
- If the pull request is approved, take the ticket out of review state
- Assign JIRA Ticket back to the developer

### CODE REVIEW

File: src/condor_utils/test_compat_classad.cpp [hunk 1]
```diff
@@ -435,7 +431,7 @@ bool test_EvalStringCharStar(ClassAd *c1, ClassAd *c2, int verbose)
 {
     bool passed = false;
     bool passedTest[4];
-    int esRetVal[2]; //evalstring return value
+    int esRetVal[2]; //evalstd::string return value
```
Comment (JaimeFrey) [inline src/condor_utils/test_compat_classad.cpp:434]: How about we only add std:: in actual code? :-)

File: src/condor_utils/test_compat_classad.cpp [hunk 2]
```diff
@@ -1169,27 +1165,27 @@ bool correctRefs(classad::References* ref, std::vector<string> expected)
...
-    bool singleAttr = (listString.find(" ") == string::npos);
+    bool singleAttr = (listString.find(" ") == std::string::npos);
 
     if(!singleAttr)
     {
     
-        string word;
-        istringstream iss(listString, istringstream::in);
+        std::string word;
+        std::istringstream iss(listString, std::istringstream::in);
```
Comment (JaimeFrey) [inline src/condor_utils/test_compat_classad.cpp:1188]: Here are more instances to fix.

File: src/condor_utils/test_compat_classad.cpp [hunk 3]
```diff
@@ -599,7 +595,7 @@ bool test_EvalStringStdString(ClassAd *c1, ClassAd *c2, int verbose)
 {
     bool passed = false;
     bool passedTest[4];
-    int esRetVal[2]; //evalstring return value
+    int esRetVal[2]; //evalstd::string return value
```
Comment (JaimeFrey) [inline src/condor_utils/test_compat_classad.cpp:598]:
```suggestion
    int esRetVal[2]; //evalstring return value
```

File: src/condor_utils/test_compat_classad.cpp [hunk 4]
```diff
@@ -609,7 +605,7 @@ bool test_EvalStringStdString(ClassAd *c1, ClassAd *c2, int verbose)
     classad::AttrList::iterator itr;
     itr = c1->begin();
 
-	std::string tmpValue; 
+	std::std::string tmpValue;
```
Comment (JaimeFrey) [inline src/condor_utils/test_compat_classad.cpp:608]:
```suggestion
	std::string tmpValue; 
```

File: src/condor_utils/test_compat_classad.cpp [hunk 5]
```diff
@@ -1107,13 +1103,13 @@ classad::References* gir_helper(classad::ClassAd* c, string attr, int verbose, b
...
 //{{{ correctRefs
-bool correctRefs(classad::References* ref, std::vector<string> expected)
+bool correctRefs(classad::References* ref, std::std::vector<std::string> expected)
 {
     bool passed = false;
 
     classad::References::iterator itr;
     classad::References::iterator itr2;
-    std::vector<string>::iterator vecItr;
+    std::std::vector<std::string>::iterator vecItr;
```
Comment (JaimeFrey) [inline src/condor_utils/test_compat_classad.cpp:1112]:
```suggestion
    std::vector<std::string>::iterator vecItr;
```

**Review by JaimeFrey — CHANGES_REQUESTED** [review]
It appears a lot of this code is never compiled.

**Comment by GregThain** [comment]
As per slack conversation, `test_compat_classad` isn't in the test suite, isn't compiled by default, and has a bunch of compiler errors even before this change.  Agreed to just git rm it.

**Review by JaimeFrey — APPROVED** [review]
LGTM

**Pull request merged**